runtime/interface.go

```diff
@@ -84,10 +84,9 @@ type Interface interface {
 	EmitEvent(cadence.Event) error
 	// GenerateUUID is called to generate a UUID.
 	GenerateUUID() (uint64, error)
-	// GetComputationLimit returns the computation limit. A value <= 0 means there is no limit
-	GetComputationLimit() uint64
-	// SetComputationUsed reports the amount of computation used.
-	SetComputationUsed(used uint64) error
+	// MeterComputation is a callback method for metering computation, it returns error
+	// when computation passes the limit (set by the environment)
+	MeterComputation(operationType common.ComputationKind, intensity uint) error
```

Comment:
hmm, was thinking most of the time this number might be really small, so allocating uint64 might be unnecessary, but we could change it in the FVM side whenever needed, thoughts?

Reply:
My reasoning was that the final value for execution effort (computation) would actually be a UFix64 so that it can be sent to the FlowFees smart contract. 

But like you have pointed out the conversion can happen in the FVM